### ISSUE TITLE
multiple buttons

### ISSUE THREAD

**Issue by 25HoursaDay** [ReedGraf/isd279games#2]
is there a way to add multiple buttons to one page? I cannot do it because of the variables interfering with each other.

**Comment by ReedGraf** [comment]
I'd love to help however I'm now so sure what you are talking about. Which buttons?

**Comment by 25HoursaDay** [comment]
The buttons you click when going to the about:blank tab, the purple ones

**Comment by ReedGraf** [comment]
Yes it is possible, I'll update the [anti-pointer-lock-bypass](https://github.com/ReedGraf/anti-pointer-lock-bypass) with a V1.1 with that feature. I'll reply to this when it has released. 👍🏼
